docs/analysis_plan.qmd

```diff
@@ -109,4 +111,9 @@ We will plot the partial effects as a function of the number of sites, populatio
 
 The estimated $\beta$ reflects the adjusted additive impact of incorporating wastewater on forecast performance taking into account the other confounding variables. We will implement this using a linear-link function in the R package `brms`. These exploratory analyses will be used to generate hypotheses into the impact of the characteristics of wastewater surveillance system on the performance of the `wwinference` model.
 
+## Extensions to consider:
+1. Evaluation of trend categorisation: We could classify each posterior draw of each forecast as increasing, decreasing, or stable based on the trajectory of hospital admissions, and so each forecast we could assign a probability for each of the three categories.
+We could then evaluate compared to the later observed data using the Brier score, and compare the Brier scores across different strata (locations, forecast dates, overall).
+2. Expansion to flu and RSV: In both cases, hospital admissions and wastewater data are available. We could consider expanding this analysis to these other pathogens, which would require GI estimates, infection -> hospital admissions delay estimates, and shedding kinetics trajectories (though we could assume similar to COVID). However, the wastewater data for flu and RSV measure subtypes A and B separately, whereas the hospital admissions does not specify subtype. The current `wwinference` model does not yet support multiple strains in the wastewater data, and so we would need to separately forecast A and B and aggregate, which wouldn't be possible without separate counts of admissions.
```

Comment:
this sounds like quite a lot of work and something for the discussion vs somethng to actually consider doing